src/build/app/settings.rs

```diff
@@ -689,6 +692,83 @@ pub enum AppSettings {
     /// [`subcommands`]: crate::App::subcommand()
     DeriveDisplayOrder,
 
+    /// Specifies that the parser should match first argument as [`subcommands`],
+    /// treat any matches as if the subcommand had been passed,
+    /// and print arguments like the [`subcommands`] are the top-level commands,
+    /// but if none match parse as usual.
+    ///
+    /// This is desired behaviour for "multicall" executables such as busybox
+    /// which dispatch to which applet to run
+    /// based on the name the executable has been linked as.
```

Comment:
Nit: I feel like the one line summary is a bit long, making it harder for people to find and use your work.

What if we had something like
```rust
/// Parse the bin name (argv[0]) as a subcommand
///
/// Busybox is a common example of a "multicall" executable where the command `cat` is a link to the `busybox` bin and, when `cat` is run, `busybox` acts is if you ran `busybox cat`.
```

Reply:
I can understand; I still don't like mine :)